MANIFEST.in

```diff
@@ -1,2 +1,23 @@
-prune artwork
+include .coveragerc .travis_before_install *.rst *.txt *.py
```

Comment:
I do not think that one needs `.coveragerc` and especially  `.travis_before_install`